crowd-server/crowd_server/apps/image_label/views.py

```diff
@@ -1,6 +1,28 @@
-from django.shortcuts import render
-from django.http import JsonResponse
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
 
+from .serializers import ImageSerializer, ImageLabelSerializer, ImageCategorySerializer
+from .models import ImageCategory, Image
 
-def test(request):
-    return JsonResponse({'message' : 'Image Label Verification Project Starting ...'})
+
+class ImageCategoryView(APIView):
+    def get(self, request):
+        categories = ImageCategory.objects.all()
+        serializer = ImageCategorySerializer(categories, many=True)
+        return Response(serializer.data)
+
+
+class ImageView(APIView):
+    def get(self, request):
+        image = Image.objects.filter(answer_count__lt=3).last()
+        serializer = ImageSerializer(image)
+        return Response(serializer.data)
+
+
+class ImageLabelView(APIView):
```

Comment:
If you want only authenticated users, could interact with your app, use the permission policy (IsAuthenticated)